.github/workflows/lambda-build-node.yaml

```
@@ -5,7 +5,7 @@ on:
         description: "The Node.js version to use"
         type: string
         required: false
-        default: 20
+        default: '20'
```

```suggestion
        default: "20"
```

to be consistent with other strings..